app/models/image_content_updater.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class ImageContentUpdater
+    def update(args)
+      ids = args == 'all' ? Affiliate.all.ids : args.split
+      update_image_content(ids)
+    end
+  
+    private
+  
+    def update_image_content(ids)
+      updated_list = []
+      failed_list = []
+  
+      Affiliate.where(id: ids).find_each do |affiliate|
+        update_affiliate_image_content(affiliate)
+        updated_list << affiliate.id
+      rescue StandardError => e
+        failed_list << { affiliate_id: affiliate.id, reason: e.inspect }
+      end
+  
+      log_completion(updated_list, failed_list)
+      [updated_list, failed_list]
+    end
+  
+    def update_affiliate_image_content(affiliate)
+      migrate_paperclip_image_to_active_storage(affiliate)
+      update_alt_text(affiliate)
+      affiliate.save
+    end
+  
+    def migrate_paperclip_image_to_active_storage(affiliate)
+      return if affiliate.mobile_logo_file_name.blank?
+  
+      logo_url = affiliate.send(:mobile_logo).url.sub(Rails.env, 'production').gsub(' ', '%20')
```

Comment:
@krbhavith This seems unnecessarily complex.  What stops us from doing `affiliate.mobile_logo.url`?

Reply:
Right we can use `affiliate.mobile_logo_url`